research/object_detection/core/box_list_ops.py

```diff
@@ -290,9 +291,10 @@ def matched_iou(boxlist1, boxlist2, scope=None):
     areas1 = area(boxlist1)
     areas2 = area(boxlist2)
     unions = areas1 + areas2 - intersections
+    eps = 1e-7
     return tf.where(
         tf.equal(intersections, 0.0),
-        tf.zeros_like(intersections), tf.truediv(intersections, unions))
+        tf.zeros_like(intersections), tf.truediv(intersections, tf.maximum(unions, eps)))
```

Comment:
Can you directly use the global constant EPSILON here? Same for other eps you added.